garak/cli.py

```diff
@@ -38,6 +68,7 @@ def main(arguments=None) -> None:
         prog="python -m garak",
         description="LLM safety & security scanning tool",
         epilog="See https://github.com/NVIDIA/garak",
+        allow_abbrev=False,
```

Comment:
Since this PR adds support for all _plugin_types_ to offer an options json or file the parser needs to disambiguate `--buffs` from `--buffs_option_file` and `--buffs_options`.

This also exposed a test that relied on abbreviation that has been update in the PR as well.